tracer/src/Datadog.Trace.BenchmarkDotNet/Datadog.Trace.BenchmarkDotNet.csproj

```
@@ -10,7 +10,7 @@
 
   <!-- For VS testing purposes only, copy all implementation assemblies to the
        to test the loading of managed profiler assemblies from disk -->
-  <PropertyGroup Condition="'$(BuildingInsideVisualStudio)'=='true' and ('$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netcoreapp3.1')">
+  <PropertyGroup Condition="'$(BuildingInsideVisualStudio)'=='true' and ('$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netcoreapp3.1' OR '$(TargetFramework)' == 'net6.0')">
```

Some future proofing
```suggestion
  <PropertyGroup Condition="'$(BuildingInsideVisualStudio)'=='true' and ('$(TargetFrameworkIdentifier)' != '.NETFramework')">
```